demos/list.html

```diff
@@ -306,6 +306,71 @@ <h3>Graphic Example - Icon with Text (Dark)</h3>
               </li>
             </ul>
           </section>
+          <section>
+            <h3>Leading Checkbox</h3>
+            <ul class="mdc-list mdc-list--non-interactive demo-list" id="leading-checkbox-list">
+              <li class="mdc-list-item checkbox-list-ripple-surface">
+                <span class="mdc-list-item__graphic">
+                  <div class="mdc-checkbox">
+                    <input type="checkbox"
+                           class="mdc-checkbox__native-control"
+                           id="leading-checkbox-blueberries"/>
+                    <div class="mdc-checkbox__background">
+                      <svg class="mdc-checkbox__checkmark"
+                           viewBox="0 0 24 24">
+                        <path class="mdc-checkbox__checkmark__path"
```

Comment:
Replace `__path` with `-path` in this entire file (it was renamed in #2096).

Otherwise, the checkboxes look like this:

![pr 2066 capture 2](https://user-images.githubusercontent.com/409245/35460544-4204dbc6-0299-11e8-979f-04350f30d39c.gif)